docs/orchestrating-elastic-stack-applications/elasticsearch/remote-clusters.asciidoc

```diff
@@ -95,23 +95,13 @@ kind: Elasticsearch
 metadata:
   name: cluster-two
 spec:
+  transport:
+    tls:
+      certificateAuthorities:
+        configMapName: remote-certs
```

Comment:
I just realised that the example needs more adjustment will update in a little bit.